device_attributes/index.bs

```diff
@@ -0,0 +1,200 @@
+<pre class='metadata'>
+Title: Device Attributes API
+Shortname: device-attributes
+Level: 1
+Editor: Anqing Zhao 132057, Google LLC https://www.google.com, anqing@google.com
+Status: w3c/CG-DRAFT
+URL: https://wicg.github.io/WebApiDevice/device_attributes
+Abstract: This document defines a web platform API that enables developers to query device information (device ID, serial number, location, etc) from managed devices. This API is invaluable for such use cases as Virtual Desktop Infrastructure (VDI) and Context-based Configuration.
+Repository: https://github.com/WICG/WebApiDevice
+Group: wicg
+</pre>
+ 
+<pre class=link-defaults>
+spec:html; type:interface; text:Navigator
+spec:url; type:dfn; text:origin
+spec:html; type:dfn; for:/; text:global object
+spec:webidl; type:dfn; text:resolve
+spec:webidl; type:dfn; text:record
+</pre>
+ 
+<pre class="anchors">
+ 
+</pre>
+ 
+# Model # {#model}
+ 
+## Managed devices ## {#managed-devices}
+ 
+The API is presumed to be used on devices which are not fully controlled by the end user, but rather by an external entity, the device administrator. Device administrators are given the power to fully control the managed devices.
```

Comment:
To let other parts of this spec refer to device administrators:

```suggestion
The API is presumed to be used on devices which are not fully controlled by the end user, but rather by an external entity, the <dfn>device administrator</dfn>. Device administrators are given the power to fully control the managed devices.
```

Reply:
Done. I also changed some 'administrator' to 'device administrator' with the hyperlink.